.github/workflows/build.yml

```diff
@@ -5,20 +5,34 @@ on: pull_request
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
 
     steps:
-      - uses: percona-platform/checkout@v2
+      - name: Checkout code
+        uses: percona-platform/checkout@v2
 
-      - name: Build grafana-dashboards
+      - name: Run with Node 16
         uses: percona-platform/setup-node@v2
         with:
-          node-version: "16"
-          cache: "npm"
-          cache-dependency-path: pmm-app/package-lock.json
+          node-version: ${{ matrix.node-version }}
```

Comment:
I don't think there is a benefit of using a matrix here, as we probably won't test different versions. Maybe just use the value as before?

Reply:
Sure thing, just changed it 👍